quinn-proto/src/congestion/cubic.rs

```diff
@@ -179,6 +181,7 @@ impl Controller for Cubic {
 
         self.recovery_start_time = Some(now);
 
+        // clippy doesn't like the duplicate line, but it would change behaviour to move it
```

Comment:
nit: has the false positive been reported to clippy?

Reply:
it has not. can I get a second opinion on whether clippy is right here? I'd rather not cite clippy as having an error to the maintainers only for them to fix my code for me.

Reply:
I see there's actually already a report linked--not sure how I missed that.